<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Christopher Antila</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">vis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs_main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>501</width>
        <height>261</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabs_settings">
          <property name="minimumSize">
           <size>
            <width>472</width>
            <height>402</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_usual_settings">
           <attribute name="title">
            <string>Usual Settings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox_heedQuality">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>81</height>
               </size>
              </property>
              <property name="title">
               <string>Interval Quality</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QRadioButton" name="rdo_heedQuality">
                 <property name="text">
                  <string>Display</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdo_ignoreQuality">
                 <property name="text">
                  <string>Ignore</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_Octaves">
              <property name="minimumSize">
               <size>
                <width>201</width>
                <height>87</height>
               </size>
              </property>
              <property name="title">
               <string>Octaves</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QRadioButton" name="rdo_compoundIntervals">
                 <property name="text">
                  <string>Compound Intervals</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdo_simpleIntervals">
                 <property name="text">
                  <string>Simple Intervals</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxwhichNs">
              <property name="minimumSize">
               <size>
                <width>191</width>
                <height>71</height>
               </size>
              </property>
              <property name="title">
               <string>Which Values of n?</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>n = </string>
                   </property>
                   <property name="buddy">
                    <cstring>line_n</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="line_n">
                   <property name="text">
                    <string notr="true">2</string>
                   </property>
                   <property name="maxLength">
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_setN">
                   <property name="text">
                    <string>Set</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_LilyPond">
              <property name="minimumSize">
               <size>
                <width>211</width>
                <height>81</height>
               </size>
              </property>
              <property name="title">
               <string>LilyPond</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QRadioButton" name="rdo_yesScore">
                 <property name="text">
                  <string>Produce Score</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdo_noScore">
                 <property name="text">
                  <string>Do not Produce Score</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_obscure_settings">
           <attribute name="title">
            <string>Obscure Settings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_Offsets">
              <property name="title">
               <string>Offsets</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Offset between Intervals:</string>
                   </property>
                   <property name="buddy">
                    <cstring>line_intervalOffset</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="line_intervalOffset">
                   <property name="text">
                    <string notr="true">0.5</string>
                   </property>
                   <property name="maxLength">
                    <number>6</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_setOffset">
                   <property name="text">
                    <string>Set</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_input">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Filenames to Analyze.
Please include one file name or directory per line.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_chooseFiles">
          <property name="text">
           <string>Choose Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_chooseDirectories">
          <property name="text">
           <string>Choose Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txt_filenames">
          <property name="documentTitle">
           <string notr="true"/>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_analyze">
          <property name="text">
           <string>Analyze!</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_output">
       <attribute name="title">
        <string>Show</string>
       </attribute>
       <widget class="QPlainTextEdit" name="txt_results">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>391</height>
         </rect>
        </property>
        <property name="plainText">
         <string notr="true">Beep</string>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
